lib/constant.go

```diff
@@ -1,9 +1,37 @@
 package sebak
 
-import "boscoin.io/sebak/lib/common"
+import (
+	"time"
+
+	"boscoin.io/sebak/lib/common"
+)
 
 const (
 	// BaseFee is the default transaction fee, if fee is lower than BaseFee, the
 	// transaction will fail validation.
 	BaseFee sebakcommon.Amount = 10000
 )
+
+var (
+	// TimeoutProposeNewBallot works when the consensus process is finished for
+	// a proposed `Ballot`, proposer will wait for `TimeoutProposeNewBallot` and
+	// then it will propose new `Ballot`.
+	TimeoutProposeNewBallot time.Duration = time.Second * 2
```

Comment:
That's the maximum time right ?

Reply:
@Geod24 not max. Proposer will wait to propose new ballot for `TimeoutProposeNewBallot` or `TimeoutProposeNewBallotFull`.

Reply:
I thought this had to be adjusted to prevent nodes round timing drifting ?

Reply:
@Geod24 This PR is based the previous design of ISAAC, so the new ISAAC concept will be different.